spec/integration/api/v1/ingredients_spec.rb

```diff
@@ -168,6 +168,42 @@
     end
   end
 
+  path '/alert-ingredients' do
+    let(:jumbo_client) { instance_double('JumboClient', products_by_query: products) }
+
+    parameter name: :user_email, in: :query, type: :string
+    parameter name: :user_token, in: :query, type: :string
+
+    get 'Show alerted ingredients' do
+      consumes 'application/json'
+      produces 'application/json'
+      description 'Retrieves ingredients with its quantity below minimum'
+
+      response '200', 'ingredients retrieved' do
+        schema(
+          type: "object",
+          properties: {
+            data: {
+              type: "array",
+              items: { "$ref" => "#/definitions/provider_ingredient" }
+            }
+          },
+          required: [
+            :data
+          ]
+        )
+
+        run_test!
+      end
```

Comment:
creo que falta un test aquí o en el `ingredient_spec.rb` para el scope

onda crear varios ingredientes que estén bajo la regla, y otros que no, y hacer un test como

`expect(descibed_class.below_minimum.count).to eq(below_minimum_quantity)`